test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/mq_and_spaces_in_metadata_value.test

```diff
@@ -0,0 +1,101 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This new .test doesn't seem related with DateTime support... maybe I'm missing something?

Reply:
Ah, true, this is an old functest I found in my cloned repo, from some older work.
It got added by mistake (I had many many FTs modified so I did: `git add test/func*/cases/*/*.test`).

Why don't you take a look at it and tell me if you're interested in keeping it. If not, I'll just remove it.
I think it was forgotten in some old PR

Reply:
I'll have a closer look but I think it can be kept. I undestand it would make stronger the regression on old issue #1156 

NTC